packages/network-controller/jest.config.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+const path = require('path');
+const merge = require('deepmerge');
+const baseConfig = require('../../jest.config.packages');
+
+const displayName = path.basename(__dirname);
+const overwriteArrays = (_destinationArray, sourceArray, _options) =>
+  sourceArray;
+
+module.exports = merge(
+  baseConfig,
+  {
+    // The display name when running multiple projects
+    displayName,
+
+    // An object that configures minimum threshold enforcement for coverage results
+    coverageThreshold: {
+      global: {
+        branches: 71.11,
+        functions: 78.57,
+        lines: 85.1,
+        statements: 85.1,
+      },
+    },
+
+    // Currently the tests for NetworkController have a race condition which
+    // causes intermittent failures. This seems to fix it.
+    testEnvironment: 'jsdom',
+    testEnvironmentOptions: {},
+  },
+  { arrayMerge: overwriteArrays },
```

Comment:
Same comment as above.

Reply:
Removed in [1b24bc7](https://github.com/MetaMask/controllers/pull/831/commits/1b24bc7fe4d12f09e6ba521383d23f723c9dc006).